src/pool.rs

```diff
@@ -41,6 +41,12 @@ impl Bundle {
             bail!("you must specify a user");
         }
 
+        // Configure connection parameters for AWS DSQL
```

Comment:
It seems like we've a mixed bad between settings here and in workloads/mod.rs. I'll merge, then unify. Also, we want Direct whenever possible.